tests/test_reading_api.py

```diff
@@ -35,7 +35,7 @@ def test_get_not_found(mocked_client):
     )
     mocked_client.return_value.get_judgment_xml.assert_called_with("bad_uri")
     assert response.status_code == 404
-    assert "Resource not found." in response.text
+    assert "No resource with that name" in response.text
```

Comment:
I couldn't find a reference to this in the source code changes. Is it that `No resource with that name` and `"Resource not found."` were already both expected within the text but you decided this string was better to assert against?